lib/components/DateRangeInput.js

```diff
@@ -46,16 +47,17 @@ const DateRangeInput = React.forwardRef(
         )}
         <Input
           ref={ref}
+          allowSingleDayRange={true}
           minDate={minDate}
           maxDate={maxDate}
-          formatDate={(date) => moment(date).format(DATE_FORMAT)}
-          parseDate={(inputStr) =>
-            inputStr
-              ? moment(inputStr, DATE_FORMAT, true).isValid()
-                ? moment(inputStr, DATE_FORMAT, true)
-                : false
-              : null
-          }
+          formatDate={date => moment(date).format(DATE_FORMAT)}
+          parseDate={inputStr => {
+            if (either(isNil, isEmpty)(inputStr)) {
+              return null;
+            }
+            const formattedDate = moment(inputStr, DATE_FORMAT, true);
```

Comment:
@yedhin _a Can you provide else condition to improve readablity.

Reply:
Using an `else` condition defeats the purpose of having a guard clause. If the expanded braces is what is causing the readability issue, then let's remove it. Also with new variable `isNotPresent`, it looks better.

If you'd like to read more about this: [Guard Clauses](https://qvault.io/2020/09/06/guard-clauses-how-to-clean-up-conditionals/)